eth/db/__init__.py

```diff
@@ -1,16 +1,10 @@
 import os
-from typing import (
-    Any,
-    cast,
-    Type
-)
-
-from eth.utils.module_loading import (
-    import_string,
-)
-from eth.db.backends.base import (
-    BaseAtomicDB,
-)
+from typing import Any, Type, cast
```

Comment:
Multiple imports must be on individual lines as per our style guide https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#formatting